python/ray/actor.py

```diff
@@ -536,23 +529,29 @@ def _remote(self,
                 creation_args = cross_language.format_args(
                     worker, args, kwargs)
             else:
-                function_signature = meta.method_signatures["__init__"]
+                function_signature = meta.method_meta.signatures["__init__"]
                 creation_args = signature.flatten_args(function_signature,
                                                        args, kwargs)
             actor_id = worker.core_worker.create_actor(
-                meta.language, meta.actor_creation_function_descriptor,
-                creation_args, meta.max_reconstructions, resources,
-                actor_placement_resources, is_direct_call, max_concurrency,
-                detached, is_asyncio)
+                meta.language,
+                meta.actor_creation_function_descriptor,
+                creation_args,
+                meta.max_reconstructions,
+                resources,
+                actor_placement_resources,
+                is_direct_call,
+                max_concurrency,
+                detached,
+                is_asyncio,
+                extension_data=str(actor_method_cpu))
```

Comment:
document: `Store actor_method_cpu in actor handle's extension data.`

Reply:
Updated.